packages/data-stores/src/queries/use-has-3rd-party-cookies.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+
+let result: boolean;
+
+export function useHas3PC() {
+	const [ hasCookies, setHasCookies ] = useState( Boolean( result ) );
+
+	useEffect( () => {
+		const iframe = document.createElement( 'iframe' );
+
+		function handler( event: MessageEvent ) {
+			const { data } = event;
+			if ( data.type === 'widgets.wp.com-cookie-check' ) {
+				setHasCookies( data.result );
+				window.removeEventListener( 'message', handler );
+				iframe.remove();
+			}
+		}
+		if ( result === undefined ) {
```

Comment:
Forgive me if I'm wrong, but it seems that `result` will always be `undefined`? Since we never actually mutate it since its declaration above.

Reply:
Hey @tyxla, yes in this case `result` is always undefined. But we addressed this right after deploying this in [this PR](https://github.com/Automattic/wp-calypso/pull/63344/commits/d385d86fc59d66e895b3cc93d376536c01058f97#diff-4aab8aca3b043d46753afae29cb0ff11e97450ba30df4003cfe0e1f4e1d7a535R14).

Reply:
Cool! I must have been looking at a stale `trunk`.